test/test_datapipe.py

```diff
@@ -52,6 +52,7 @@
     runtime_validation,
     runtime_validation_disabled,
 )
+from torch.utils.data.graph import traverse
```

Comment:
Side note: Do you think we can also add this to one of the standard tests for each DataPipe in the future?
Maybe open an issue?

Reply:
I think it should go here:
https://github.com/pytorch/pytorch/wiki/DataPipes-testing-requirements

One of these two?
- Test if DataPipes creates properly linked DataPipes graph
- Test if DataPipe is picklable (using pickle or dill)

Reply:
I mean we need to test all existing and in-development DataPipes for this case.

Reply:
After iteration, it's still picklable.

Reply:
I see, I will add it to the testing requirement page. 

Perhaps we should have an issue to ensure all DataPipes have tests that meet all the requirements on that page? Or do you prefer to have a specific issue about being picklable after iteration for now?
I can do either.

Reply:
I would go with former one and categorize which test is missing for each DataPipe